realtimenet/display.py

```diff
@@ -133,6 +134,47 @@ def display(self, img, display_data):
         return img
 
 
+class DisplayFPS(BaseDisplay):
+    """
+    Display camera fps and inference engine fps on debug window.
+    """
+
+    def __init__(self, neural_network, y_offset=20):
+        super().__init__(y_offset)
+        self.net = neural_network
+
+        self.fps_update_rate = 0.1
```

Comment:
I think we can call it `self.update_rate` now that we have a specific class for it